### PR TITLE
Towards #3527

### PR DESCRIPTION
**Pull request by sfilipi**
Addresses the typos documented on #3527

### CODE REVIEW

File: src/Microsoft.ML.Mkl.Components/OlsLinearRegression.cs [hunk 1]
```diff
@@ -33,7 +33,7 @@ namespace Microsoft.ML.Trainers
 {
     /// <summary>
     /// The <see cref="IEstimator{TTransformer}"/> for training a linear regression model using
-    /// ordinary least squares (OLS) for estimating the parameters of the linear regression model.
+    /// <a href="https://en.wikipedia.org/wiki/Ordinary_least_squares">ordinary least squares (OLS)</a> for estimating the parameters of the linear regression model.
```
Comment (artidoro) [inline src/Microsoft.ML.Mkl.Components/OlsLinearRegression.cs]: I remember some links being broken when there were parentheses. 
If it is confirmed to be an issue maybe we could omit (OLS) from the link.

File: src/Microsoft.ML.StandardTrainers/Standard/Online/OnlineGradientDescent.cs [hunk 2]
```diff
@@ -25,7 +25,7 @@ namespace Microsoft.ML.Trainers
 {
     /// <summary>
     /// The <see cref="IEstimator{TTransformer}"/> for training a linear regression model using
-    /// Online Gradient Descent (OGD) for estimating the parameters of the linear regression model.
+    /// <a href="https://en.wikipedia.org/wiki/Gradient_descent">Online Gradient Descent (OGD)</a> for estimating the parameters of the linear regression model.
```
Comment (artidoro) [inline src/Microsoft.ML.StandardTrainers/Standard/Online/OnlineGradientDescent.cs]: Same comment here. But again, it might not be an issue.

Comment (sfilipi) [reply src/Microsoft.ML.StandardTrainers/Standard/Online/OnlineGradientDescent.cs]: don't think we have issues with `<a href>` inside xml. 
Probably markdown

File: src/Microsoft.ML.Mkl.Components/OlsLinearRegression.cs (hunk 1 above)
Comment (sfilipi) [reply src/Microsoft.ML.Mkl.Components/OlsLinearRegression.cs]: Should not be an issue for `<a>` inside xml.

**Pull request merged**